model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -1283,6 +1283,7 @@ async def execute(
             min_workers=request.min_workers,
             max_workers=request.max_workers,
             endpoint_type=request.endpoint_type,
+            sync_can_scale_from_zero=self.model_endpoint_service.can_autoscale_sync_stream_endpoints_from_zero(),
```

Comment:
why sync only?

Reply:
I can rename this if you want, I think my thinking was "the variable name is getting kinda long"

Reply:
maybe we just remove `sync` and `stream` in these? and just have like a comment/disclaimer that async isn't implemented/supported?

Reply:
async already scales from zero (that was one of its advantages before)

Reply:
oh i guess that makes sense. I think we can still remove the sync/stream from the variable name

Reply:
wait actually think "http_scale_from_zero" seems reasonable?

Reply:
lol until we have to support websockets XP

i'm fine with it though